extensions/smallrye-reactive-messaging-rabbitmq/runtime/pom.xml

```diff
@@ -30,6 +30,10 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-vertx</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-credentials</artifactId>
```

Comment:
should it be optional?

Reply:
This is a very small extension that only adds `CredentialsProvider` and `CredentialsProviderFinder`. I'm following all of the other extensions' lead here (e.g. datasource, atrial, reactive-pg-client, etc.), they all use it without declaring it as optional.

Reply:
Agreed.